stock_indicators/_cstypes/datetime.py

```
@@ -20,14 +19,55 @@
+    hour = cs_datetime.Hour
+    minute = cs_datetime.Minute
+    second = cs_datetime.Second
+
+    # Microseconds within the second from ticks (1 tick = 100 ns)
+    microsecond = int((cs_datetime.Ticks % TICKS_PER_SECOND) // 10)
+
+    # Attach tzinfo only for UTC
+    if str(cs_datetime.Kind) == 'Utc':
+        return PyDateTime(year, month, day, hour, minute, second, microsecond, tzinfo=PyTimezone.utc)
+    else:
+        return PyDateTime(year, month, day, hour, minute, second, microsecond)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)